### ISSUE TITLE
Unexpected API error. KeyError: com.github.isacikgoz.ukill

### ISSUE THREAD

**Issue by damianchoc** [isacikgoz/ukill#3]
Error,  extension dont work

**Comment by isacikgoz** [comment]
Thanks for the feedback @damianchoc  I've got the same error when I try to install. First, it said "request time out" then same error as yours with a "extension already running error"

Can you check if it is installed somehow?

**Comment by damianchoc** [comment]
yeah its installed. tbh ive got that error in some of ulauncher extensions. Some work and some have that error and don't work. I don't know why. Its bad cuz ulauncher is cool, but without extensions its not usefull, eh

**Comment by isacikgoz** [comment]
It think this issue is related with [Ulauncher](https://github.com/Ulauncher/Ulauncher) itself. Maybe you can open an issue there and link with this one. Since I cannot reproduce it, I lack the knowledge about this issue.
